cms/templates/partials/news-and-events-carousel.html

```diff
@@ -0,0 +1,25 @@
+{% load image_version_url wagtailcore_tags %}
+<div id="news-and-events" class="news-and-events-block">
+  <div class="container">
+    <div class="head">
+      <h1>{{ page.heading }}</h1>
+    </div>
+    <div class="news-and-events-slider">
+      {% for card in page.items %}
+      <div class="slide">
+        <div class="slide-holder">
+          {% if card.value.image %}
+            <img src="{% image_version_url card.value.image "fill-330x200" %}" alt="{{ card.value.name }}" width="330" height="200" loading="lazy"/>
+          {% endif %}
+          <div class="card-type">
+            {{ card.value.content_type }}
```

Comment:
are we considering to keep these values required in CMS ? I just mean, if no any data then we may have empty HTML tags here.

Reply:
All fields are mandatory right now, according to the requirements.